Packages/manifest.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "com.neengash.unityfeto": "https://github.com/Neengash/UnityFeTo.git?path=/Assets/FeTo",
```

Comment:
It's better to use this url here (As told in the readme u.u"):
https://github.com/Neengash/UnityFeTo.git#FeTo